site/src/components/accordion.md

```diff
@@ -0,0 +1,137 @@
+classType: Component
+classPrefixes:
+  - accordion
+---
+
+# Accordion
+
+The Accordion component contains a disclosure widget, where information is only visible when the element is toggled to an "open" state. A Summary or label must be provided using the `<summary>` element.
+
+## Usage
+
+Here is an example of a standard `.accordion` usage. By default `<details>` elements are closed on page load and toggle open or closed when the user clicks the `<summary>`.
+
+```html
+<details class="accordion">
+	<summary>Accordion header 1</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 1</p>
+	</div>
+</details>
+<details class="accordion">
+	<summary>Accordion header 2</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 2</p>
+	</div>
+</details>
+<details class="accordion">
+	<summary>Accordion header 3</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 3</p>
+	</div>
+</details>
+```
+
+## Attributes
+
+### Open
+
+There is an option to have the accordion component load in the expanded state by adding the `open` attribute.
+
+```html
+<details class="accordion" open>
+	<summary>Accordion header 1</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 1</p>
+	</div>
+</details>
+<details class="accordion">
+	<summary>Accordion header 2</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 2</p>
+	</div>
+</details>
+<details class="accordion">
+	<summary>Accordion header 3</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 3</p>
+	</div>
+</details>
+```
+
+### Name
+
+Using the `name` attribute, there is an option to group multiple accordion components, which allows only one open panel at a time.
+
+```html
+<details class="accordion" name="accordion-group-1">
+	<summary>Accordion header 1</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 1</p>
+	</div>
+</details>
+<details class="accordion" name="accordion-group-1">
+	<summary>Accordion header 2</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 2</p>
+	</div>
+</details>
+<details class="accordion" name="accordion-group-1">
+	<summary>Accordion header 3</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 3</p>
+	</div>
+</details>
+```
+
+## Icon location
+
+By default the expansion indicator icon is before the summary content. Using the `.accordion-icon-end` class, the icon can be positioned at the end of the summary. The icon is absolutely positioned to remain at the top if Summary content wraps or contains multiple lines.
+
+<details class="accordion">
+  <summary>Accordion header 1</summary>
+  <div class="accordion-content">
+  	<p>Accordion panel 1</p>
+  </div>
+</details>
+<details class="accordion border-top">
+  <summary class="accordion-icon-end">Accordion header 2</summary>
```

Comment:
two things:
- can you wrap the whole thing into 'html``' so the markup example is visible. also, can remove the first (default) scenario and the `border-top` atomic
- can that class be added to the parent element so it's a little easier to read it and less things to remember where which thing should be assigned

Reply:
added the missing markup wrapper, removed the first example. The refactor of the arrow position keeps it no outer padding in both start and end positions so it can be added with atomics if desired.

![image](https://github.com/user-attachments/assets/c05105f1-da7a-4cd8-ba3d-5fd5cf86d218)